pyotgw/pyotgw.py

```diff
@@ -354,57 +354,132 @@ async def get_status(self):
+                v.DATA_SLAVE_CH_ACTIVE: int(slave_status[6]),
+                v.DATA_SLAVE_DHW_ACTIVE: int(slave_status[5]),
+                v.DATA_SLAVE_FLAME_ON: int(slave_status[4]),
+                v.DATA_SLAVE_COOLING_ACTIVE: int(slave_status[3]),
+                v.DATA_SLAVE_CH2_ACTIVE: int(slave_status[2]),
+                v.DATA_SLAVE_DIAG_IND: int(slave_status[1]),
+                v.DATA_CONTROL_SETPOINT: float(fields[1]),
+                v.DATA_REMOTE_TRANSFER_DHW: int(remote_params[0][7]),
+                v.DATA_REMOTE_TRANSFER_MAX_CH: int(remote_params[0][6]),
+                v.DATA_REMOTE_RW_DHW: int(remote_params[1][7]),
+                v.DATA_REMOTE_RW_MAX_CH: int(remote_params[1][6]),
+                v.DATA_COOLING_CONTROL: float(fields[3]),
+                v.DATA_CONTROL_SETPOINT_2: float(fields[4]),
+                v.DATA_SLAVE_MAX_RELATIVE_MOD: float(fields[5]),
+                v.DATA_SLAVE_MAX_CAPACITY: int(capmodlimits[0]),
+                v.DATA_SLAVE_MIN_MOD_LEVEL: int(capmodlimits[1]),
+                v.DATA_ROOM_SETPOINT: float(fields[7]),
+                v.DATA_REL_MOD_LEVEL: float(fields[8]),
+                v.DATA_CH_WATER_PRESS: float(fields[9]),
+                v.DATA_DHW_FLOW_RATE: float(fields[10]),
+                v.DATA_ROOM_SETPOINT_2: float(fields[11]),
+                v.DATA_ROOM_TEMP: float(fields[12]),
+                v.DATA_CH_WATER_TEMP: float(fields[13]),
+                v.DATA_DHW_TEMP: float(fields[14]),
+                v.DATA_OUTSIDE_TEMP: float(fields[15]),
+                v.DATA_RETURN_WATER_TEMP: float(fields[16]),
+                v.DATA_CH_WATER_TEMP_2: float(fields[17]),
+                v.DATA_EXHAUST_TEMP: int(fields[18]),
+                v.DATA_SLAVE_DHW_MAX_SETP: int(dhw_setp_bounds[0]),
+                v.DATA_SLAVE_DHW_MIN_SETP: int(dhw_setp_bounds[1]),
+                v.DATA_SLAVE_CH_MAX_SETP: int(ch_setp_bounds[0]),
+                v.DATA_SLAVE_CH_MIN_SETP: int(ch_setp_bounds[1]),
+                v.DATA_DHW_SETPOINT: float(fields[21]),
+                v.DATA_MAX_CH_SETPOINT: float(fields[22]),
+                v.DATA_VH_MASTER_VENT_ENABLED: int(vh_master_status[7]),
+                v.DATA_VH_MASTER_BYPASS_POS: int(vh_master_status[6]),
+                v.DATA_VH_MASTER_BYPASS_MODE: int(vh_master_status[5]),
+                v.DATA_VH_MASTER_FREE_VENT_MODE: int(vh_master_status[4]),
+                v.DATA_VH_SLAVE_FAULT_INDICATE: int(vh_slave_status[7]),
+                v.DATA_VH_SLAVE_VENT_MODE: int(vh_slave_status[6]),
+                v.DATA_VH_SLAVE_BYPASS_STATUS: int(vh_slave_status[5]),
+                v.DATA_VH_SLAVE_BYPASS_AUTO_STATUS: int(vh_slave_status[4]),
+                v.DATA_VH_SLAVE_FREE_VENT_STATUS: int(vh_slave_status[3]),
+                v.DATA_VH_SLAVE_DIAG_INDICATE: int(vh_slave_status[1]),
+                v.DATA_VH_CONTROL_SETPOINT: int(fields[24], 2),
+                v.DATA_VH_RELATIVE_VENT: int(fields[25], 2),
```

Comment:
Just noticed the `, 2` here. I will fix it soon.

Reply:
Fixed in 0b67567fa0ad22fa2cb3701a8e5472bb01ed1c09

Reply:
Actually it was there on purpose, since the docs says it is returned as a byte, which I interpreted to mean 8 bits, but I checked it now and it looks indeed just like an int field